timeline/static/timeline/js/timeline.js

```diff
@@ -61,3 +61,25 @@ $(document).ready(function() {
         });
     });
 });
+$('body').on('click', '.sodar-tl-link-status-extra-data', function () {
```

Comment:
There is now quite a bit of repetition between clicking the extra data link in two different places. Please unify the repeating bits under e.g. a separate helper function and call that instead.

Reply:
Created a helper function